scons test fails for Visual Studio 2019

### Cantera version

2.5.0a3

### Operating System

Windows 10 / Visual Studio 2019

### Python/MATLAB version

Anaconda 3.7

### Expected Behavior

`scons test` runs without error

### Actual Behavior

`scons test` fails with `error C2512:`, i.e.

```
test\general\test_units.cpp(8): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(17): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(27): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(36): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(66): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(79): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(88): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(96): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(112): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(119): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
test\general\test_units.cpp(129): error C2512: 'Cantera::UnitSystem': no appropriate default constructor available
C:\Users\<username>\Github\cantera\include\cantera/base/Units.h(100): note: see declaration of 'Cantera::UnitSystem'
scons: building terminated because of errors.
```
### Steps to reproduce
1. Compile current master (commit 674c37ce3cbfd867179d8ab6c9af605b5d56eb6c) using VS2019 (via MSVC 14.1)
2. Run `scons test`, which fails (see attached ... [scons_test.log](https://github.com/Cantera/cantera/files/3387498/scons_test.log))
3. [Details on the toolchain are posted here](https://github.com/ischoegl/cantera-toolchain/blob/master/win10-conda-VS.md)

--- a/include/cantera/base/Units.h
+++ b/include/cantera/base/Units.h
@@ -101,7 +101,11 @@ class UnitSystem
 {
 public:
     //! Create a unit system with the specified default units
-    UnitSystem(std::initializer_list<std::string> units={});
+    UnitSystem(std::initializer_list<std::string> units);
+
+    //! Default constructor for unit system (needed as VS2019 does not
+    //! recognize an optional argument with a default value)
+    UnitSystem() : UnitSystem({}) {}
 
     //! Set the default units to convert from when explicit units are not
     //! provided. Defaults can be set for mass, length, time, quantity, energy,

